pkgs/by-name/ka/karlyriceditor/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  qt6,
+  ffmpeg_4,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "karlyriceditor";
+  version = "3.3";
+
+  src = fetchFromGitHub {
+    owner = "gyunaev";
+    repo = finalAttrs.pname;
+    rev = finalAttrs.version;
+    sha256 = "sha256-i4uZtHxnreow7a5ZX6WCXMUSwgkUJS/1oDCJOgfFjHw=";
+  };
+
+  nativeBuildInputs = [
+    qt6.wrapQtAppsHook
+    qt6.qmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    ffmpeg_4
+    qt6.qtmultimedia
+  ];
+
+  installPhase =
+    if stdenv.hostPlatform.isDarwin then
+      ''
+        mkdir -p $out/Applications
+        mv karlyriceditor.app $out/Applications
+      ''
+    else
+      ''
+        install -Dm755 bin/karlyriceditor $out/bin/karlyriceditor
+        install -Dm644 packages/karlyriceditor.desktop $out/share/applications/karlyriceditor.desktop
```

Comment:
This desktop file contains `Icon=/usr/share/pixmaps/karlyriceditor.png`
Consider using `substituteInPlace` with `--replace-fail` to swap this value out to `Icon=karlyriceditor`